sdk/storage/azblob/zt_examples_test.go

```diff
@@ -112,7 +106,7 @@ func Example() {
 	// List methods returns a pager object which can be used to iterate over the results of a paging operation.
 	// To iterate over a page use the NextPage(context.Context) to fetch the next page of results.
 	// PageResponse() can be used to iterate over the results of the specific page.
-	pager := containerClient.NewListBlobsFlatPager(nil)
+	pager := client.NewListBlobsPager(containerName, nil)
 	for pager.More() {
 		resp, err := pager.NextPage(context.TODO())
 		if err != nil {
```

Comment:
Shouldn't line 113 be `handleError(err)` to be consistent with other error handling?